debian/rules

```diff
@@ -1,33 +1,14 @@
 #!/usr/bin/make -f
 
-DH_VERBOSE = 1
+export DH_VERBOSE = 1
 
-DPKG_EXPORT_BUILDFLAGS = 1
-include /usr/share/dpkg/default.mk
+# pkg-pecl.mk contains all we need
+include /usr/share/dh-php/pkg-pecl.mk
 
-export DEB_BUILD_MAINT_OPTIONS = hardening=+all
-
-%:
-	dh $@ 
-
-override_dh_auto_clean:
-	cd ./src; phpize --clean
-
-override_dh_auto_configure:
-	cd ./src; phpize
-	cd ./src; ./configure --enable-snuffleupagus
+PECL_CONFIGURE_MAINT_APPEND = --enable-snuffleupagus
```

Comment:
We can use `PECL_NAME_OVERRIDE` to rename the package to php-snuffleupagus if you want :)

Reply:
Okay